components/brave_ads/core/internal/account/account.h

```diff
@@ -59,6 +59,7 @@ class Account final : public AdsClientNotifierObserver,
 
   void SetWallet(const std::string& payment_id,
                  const std::string& recovery_seed);
+  void SetWalletFrom(const WalletInfo& wallet_info);
```

Comment:
Can we please add a comment to state this is a temporary fix until 1.54.x lands which has a more robust solution